docs/en/stack/security/index.asciidoc

```diff
@@ -55,12 +55,6 @@ corruption. However, there's nothing to stop deliberate tampering or data
 interception. The {stack-security-features} preserve the integrity of your
 data by <<ssl-tls, encrypting communications>> to and from nodes.
 
-////
-For even
-greater protection, you can increase the <<ciphers, encryption strength>> and
```

Comment:
I think the first of these two links still works?  i.e. it redirects me to https://www.elastic.co/guide/en/elasticsearch/reference/master/ciphers.html